src/components/navigation/navigation.tsx

```diff
@@ -104,99 +86,154 @@ const SocialLink = styled.a`
   }
 `;
 
-const LegalLinks = styled.div`
-  position: absolute;
-  left: 31px;
-  bottom: 33px;
+const LegalLink = styled(Link)`
+  display: block;
+
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 22px;
+
+  text-decoration: none;
+  color: #202840;
+
+  /**
+   * necessary for Safari
+   */
+  &:link {
+    color: #202840;
+  }
```

Comment:
legal links don't display alongside each other on desktop anymore. also the space between 'nav' and the top of the page links, looks too small imo?
design:
<details>
<summary>design</summary>
<img width="490" alt="Screenshot 2020-07-06 at 10 25 17" src="https://user-images.githubusercontent.com/31735645/86572368-23bdb180-bf73-11ea-9f8f-31ec13ab33bd.png">
</details>

<details>
<summary>now</summary>
<img width="1438" alt="Screenshot 2020-07-06 at 10 25 38" src="https://user-images.githubusercontent.com/31735645/86572414-31733700-bf73-11ea-91eb-d151e4b3fa79.png">
</details>